docs/user_manual.md

```diff
@@ -294,6 +297,9 @@ example `{"quic": {"internalPort": 80, "protocol": "udp"}}` or
 The name of the endpoint specified in the port mapping will be used if
 specifying service registration using the `registration` below.
 
+#### ramdisks
```

Comment:
looks to me like this is already documented in this file and more extensively https://github.com/spotify/helios/blob/8424a029eebdd3044d73f580d5e0ad74fba819fb/docs/user_manual.md#ramdisks-1

Reply:
Thanks. I moved them to the right place.